.bumpversion.cfg

```diff
@@ -18,6 +18,8 @@ serialize =
 [bumpversion:file(package_version):roles/confluent.variables_handlers/defaults/main.yml]
 
 [bumpversion:file(repo_version):roles/confluent.variables_handlers/defaults/main.yml]
+
+[bumpversion:file(repo_version):README.md]
```

Comment:
Lets update the link to relative path and get rid of this entry here.

Reply:
Done